src/assets/stylesheets/main/_api.scss

```diff
@@ -222,3 +240,9 @@ table.longtable.docutils.data.align-default {
     }
   }
 }
+
+// a rule to reset margin for code-blocks with a caption
+.md-typeset .code-block-caption + .notranslate pre,
```

Comment:
Where does this issue occur?

Reply:
anytime the code block has a caption.

Reply:
Oh okay.  Perhaps we should move this to _typeset.scss instead since it doesn't pertain to API documentation specifically?  (I intended this file for styling sphinx "object descriptions".)

Reply:
I can move it to there. I was more concerned with merging conflict from upstream...